src/styles/blocks/main.scss

```diff
@@ -0,0 +1,28 @@
+main {
+  padding: 50px 40px;
+
+  --column-counts: 1;
```

Comment:
It seems there is a typo in the variable declaration '--column-counts'. It should be '--column-count', as 'counts' implies plural, but it seems to represent a single value.